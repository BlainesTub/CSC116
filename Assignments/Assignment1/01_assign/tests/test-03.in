This is a well chosen key. The attacker should have no knowledge of how it was chosen, and of course, should have no access to it.
THis is the message to encrypt
However, one-time-pad has one weakness. The key should never be used again. In other words, this second encrypted message
and any further encrypted message makes the key weak and subject to attack.
In other words: when using one-time-pad encryption, you should only encrpyt one message per key.
