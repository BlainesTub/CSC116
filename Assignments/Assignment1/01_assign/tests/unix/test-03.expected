Original key [This is a well chosen key. The attacker should have no knowledge of how it was chosen, and of course, should have no access to it.]
Original message [THis is the message to encrypt]
Encrypted [(0) (0)(0)(0)(0)(0)(0)(21)H(18)E(1)(9)S(16)(9)(8)(22)E(26)OK(0)(23)MR-(24)(17)]
Decrypted [THis is the message to encrypt]
Original message [However, one-time-pad has one weakness. The key should never be used again. In other words, this second encrypted message]
Encrypted [(28)(7)(30)(22)V(12)(1)(12)AO(25)(0)A(24)I(14)(13)B(3)(4)(10)(0)(3)(4)(10)(14)O:(13)EW(4)(21)(31)(15)(6)(24)(22)\(0)'(0)(10)U(7)(1)YH(18)(30)(10)U(2)(11)(0)(5)(11)(25)(18)(30)E(6)(2)EU(28)(3)DH(14)(16)A(0)(26)(14)W((29)(0)(12)(28)(7)(22)(23)N[O(19)(10)(23)(12)O(18)H(10)(28)U(1)(22)(6)CN(23)H(10)(27)(15)(22)Y(24)(21)(19)(1)(0)(3)(10)S(18)(2)(4)(0)]
Decrypted [However, one-time-pad has one weakness. The key should never be used again. In other words, this second encrypted message]
Original message [and any further encrypted message makes the key weak and subject to attack.]
Encrypted [5(6)(13)SA(7)(10)(0)(7)U(5)(17)(4)(9)RC(13)(1)(16)(23)(23)P(31)(0)(29)(14)M1(27)(22)A(6)(17)T(12)(2)(0)(0)(1)(0)(7)(0)(10)U(7)(1)YH(22)(19)(4)KN(14)N(15)N(28)(2)(14)(15)(1)(4)(17)(0)(27)(9)(0)(9)(27)(3)A(10)(31)(14)]
Decrypted [and any further encrypted message makes the key weak and subject to attack.]
Original message [In other words: when using one-time-pad encryption, you should only encrpyt one message per key.]
Encrypted [(29)(6)I(28)T(1)(22)RAW(24)(23)(8)(31)(26)C(31)(7)(22)(11)NU(24)(12)(23)I(0);(6)(0)(13)(21)(29)(25)(4)N(27)(4)(22)(0)(22)(6)(12)(7)(21)(20)T(1)(14)(24)I(0)(23)(0)UK(29)(7)(24)(25)(9)(0)G(10)N(3)(31)(0)(13)(1)(20)R(25)(13)TW(14)(29)EC(5)(10)(0)(22)(15)KEA(30)(1)RO(13)E(26)A]
Decrypted [In other words: when using one-time-pad encryption, you should only encrpyt one message per key.]
